tensorflow/lite/kernels/internal/tensor_utils.cc

```diff
@@ -0,0 +1,25 @@
+/* Copyright 2017 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+#include "tensorflow/lite/kernels/internal/tensor_utils.h"
+
+#include "tensorflow/lite/kernels/internal/optimized/neon_check.h"
```

Comment:
Does this file exist in TFLM?

Reply:
https://github.com/tensorflow/tflite-micro/blob/main/tensorflow/lite/kernels/internal/optimized/neon_check.h